packages/dotcom-ui-shell/src/components/OptimizelyHead.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+const OptimizelyHead = () => {
+  return <script src="https://cdn.optimizely.com/js/18203511709.js" />
```

Comment:
When the document is loaded it will stop to parse scripts in the `<head>` element.

This means it's likely that this implementation will drastically slow down page load times.